Source/ACE/Network/Handlers/CharacterHandler.cs

```diff
@@ -124,33 +124,107 @@ public static void CharacterRestore(ClientMessage message, Session session)
             reader.Skip(4);   /* Unknown constant (1) */
             character.Heritage = reader.ReadUInt32();
             character.Gender = reader.ReadUInt32();
-            Appearance appearance = Appearance.FromNetowrk(reader);
+            Appearance appearance = Appearance.FromNetwork(reader);
 
             // pull character data from the dat file
             SexCG sex = cg.HeritageGroups[(int)character.Heritage].SexList[(int)character.Gender];
-            
+
             character.MotionTableId = sex.MotionTable;
+            character.SetDataIdProperty(PropertyDataId.MotionTable, character.MotionTableId);
```

Comment:
This should be redundant.  The MotionTableId ought to (under the hood) call SetDataIdProperty (and as far as I can tell, it does:
https://github.com/ACEmulator/ACE/blob/object-overhaul/Source/ACE.Entity/AceObject.cs, line 108

Same applies to all the properties below.